aries-core/src/js/components/helpers/Anchors/NavLink.js

```diff
@@ -3,20 +3,23 @@ import PropTypes from 'prop-types';
 import { Anchor, Text } from 'grommet';
 
 export const NavLink = ({ active, children, label, size, ...rest }) => {
-  const activeTextColor = active ? 'text' : null;
-  const defaultTextColor = 'text-xweak';
-  const fontWeight = 400;
-  const [textColor, setTextColor] = useState(defaultTextColor);
+  const textColor = 'text-strong';
+  const activeFontWeight = 700;
+  const defaultFontWeight = 400;
```

Comment:
Wondering if those should be hardcoded or part of the theme.

Reply:
I've gone back and forth, debating the approach, and but have been leaning towards more tightly coupling the style to the component (because we have 3 distinct executions) and use the theme for the basic, shared styling.

**NavLink**
![Screen Shot 2019-12-12 at 11 38 32 AM](https://user-images.githubusercontent.com/1756948/70749886-822af600-1cea-11ea-9e3c-8a0e0870f6df.png)

**AnchorCallToAction**
![Screen Shot 2019-12-12 at 11 38 27 AM](https://user-images.githubusercontent.com/1756948/70749890-86efaa00-1cea-11ea-9e0d-c623c94c7d37.png)

**"Plain" Anchor** 
![Screen Shot 2019-12-12 at 11 38 10 AM](https://user-images.githubusercontent.com/1756948/70749895-89ea9a80-1cea-11ea-8ccb-d17e23e16a4f.png)

I think the alternative (putting the styling in the theme) will require us to use anchor.extend and then style based off of a prop. When exploring this option, things start to get messy and less readable.

I landed on this approach because I felt the theme should contain our "base" or "default" anchor, then the specialized use cases can modify from there.

Reply:
Thank you for our great offline discussion, let's continue exploring the option of allowing `type` to Anchor, while type could be any of the above Anchor options above. It will make the user more resourceful while exploring the different Anchor options.

Reply:
For documentation purposes, what we discussed was the following:

- As the design system instructions get fleshed out regarding Anchor usage and behavior (i.e. which use cases should use which type of anchor), consider creating an Anchor component which has a `type` property whose values might be (callToAction, navLink, plain, etc.)
- This would allow us to present a single, consolidated component to end users, allowing them to easily select the appropriate execution (as opposed to having to browse through all potential anchors).